drivers/can/can.c

```diff
@@ -916,12 +918,15 @@ static ssize_t can_write(FAR struct file *filep, FAR const char *buffer,
  *
  ****************************************************************************/
 
-static inline ssize_t can_rtrread(FAR struct can_dev_s *dev,
-                                  FAR struct canioc_rtr_s *rtr)
+static inline ssize_t can_rtrread(FAR struct file *filep,
+                                  FAR struct canioc_rtr_s *request)
 {
-  FAR struct can_rtrwait_s *wait = NULL;
+  FAR struct can_dev_s *dev       = filep->f_inode->i_private;
+  FAR struct can_rtrwait_s *wait  = NULL;
```

Comment:
remvoe the extra space before =

Reply:
Fixed now

Reply:
but @MTres19 I mean wait variable too

Reply:
Whoops, this time it's really fixed I think @xiaoxiang781216. Thanks.